documentation/faq.md

```diff
@@ -75,4 +75,4 @@ We're full with ideas where to take Appyx further! A more detailed roadmap will
 
 ## Other
 
-Have a question? Raise it in [Discussions](https://github.com/bumble-tech/appyx/discussions)!.
\ No newline at end of file
+Have a question? Raise it in [Discussions](https://github.com/bumble-tech/appyx/discussions)!.
```

Comment:
I accidentally reformatted these files, and because they didn't have a newline character at the end, Android Studio won't stop trying to stage them, so I thought I may as well commit the change